src/main/webapp/app/exercises/shared/exercise-hint/manage/exercise-hint-detail.component.html

```diff
@@ -16,7 +16,7 @@ <h2><span jhiTranslate="artemisApp.exerciseHint.detail.title">Exercise Hint</spa
                 <dt><span jhiTranslate="artemisApp.exerciseHint.exercise">Exercise</span></dt>
                 <dd>
                     <div *ngIf="exerciseHint.exercise">
-                        <a [routerLink]="['/exercise', exerciseHint.exercise?.id, 'view']">{{ exerciseHint.exercise?.id }}</a>
+                        <a routerLink="../../../'view']">{{ exerciseHint.exercise?.id }}</a>
```

Comment:
I found one small problem here:
```suggestion
                        <a routerLink="../../../view">{{ exerciseHint.exercise?.id }}</a>
```